packages/zendesk-adapter/src/filters/guide_translation.ts

```diff
@@ -49,21 +53,50 @@ const needToOmit = (change: Change<InstanceElement>): boolean =>
   isDefaultTranslationAddition(change) || parentRemoved(change)
 
 /**
- * a different filter is needed for the default translation and translations for which the section
- * has been removed, as they are deployed during the deployment of the section itself. Therefore,
- * this filter marks these translations as successfully deployed without actually deploying them.
+ * a different filter is needed for the default translation and translations for which the parent
+ * has been removed, as they are deployed during the deployment of the parent itself. Therefore,
+ * this filter marks translations as successfully deployed without actually deploying them.
+ * For addition of default article translation, as we deploy the article with a temporary title and body,
+ * we transform the change to a modification change and deploy it in this way.
  * The rest of the translations will be deployed in the default deploy filter.
  */
-const filterCreator: FilterCreator = () => ({
+const filterCreator: FilterCreator = ({ config, client }) => ({
   name: 'guideTranslationFilter',
   deploy: async (changes: Change<InstanceElement>[]) => {
     const [translationChangesToIgnore, leftoverChanges] = _.partition(
       changes,
       needToOmit,
     )
+    // split default article translation addition changes out of changes to ignore
+    const [defaultArticleTranslationAdditionChanges, otherTranslationChanges] = _.partition(
+      translationChangesToIgnore,
+      change => getChangeData(change).elemID.typeName === ARTICLE_TRANSLATION_TYPE_NAME
+      && isDefaultTranslationAddition(change),
```

Comment:
readability nit - should always indent when breaking the implementation into multiple lines
```suggestion
      change => getChangeData(change).elemID.typeName === ARTICLE_TRANSLATION_TYPE_NAME
        && isDefaultTranslationAddition(change),
```